src/views/Restoration/RestorationReviewConfirm.vue

```diff
@@ -116,6 +126,18 @@ export default class RestorationReviewConfirm extends Vue {
     return GetCorpFullDescription(this.getEntityType)
   }
 
+  get businessOfficeEmail (): string {
+    return this.getBusinessContact.email
+  }
+
+  get applicantEmail (): string {
+    const orgPeople = this.getAddPeopleAndRoleStep.orgPeople
+    const applicants = orgPeople.filter(
+      orgPerson => orgPerson.roles.some(role => role.roleType === RoleTypes.APPLICANT)
+    )
+    return applicants[0]?.officer.email
```

Comment:
If this returns undefined then the DocumentDelivery component displays "Not Entered".